app/controllers/admins/users_controller.rb

```diff
@@ -55,10 +54,7 @@ def last_super_admin_tries_to_update_its_role?
     end
 
     def last_super_admin?
-      @last_super_admin ||= begin
-        super_admins = User.where(role: User::USER_ROLES[:super_admin]).size 
-        super_admins < 2
-      end
+      User.where(role: User::USER_ROLES[:super_admin]).size < 2
```

Comment:
@last_super_admin ||= User.where(role: User::USER_ROLES[:super_admin]).size < 2
?
CC @mmrshk

Reply:
Agree. @last_super_admin ||= User.where(role: User::USER_ROLES[:super_admin]).size < 2 is better